kivy/__init__.py

```diff
@@ -331,14 +331,19 @@ def kivy_usage():
 
         mp_fork = None
         try:
-            mp_fork = opts['multiprocessing-fork']
-        except:
+            for opt, arg in opts:
+                if opt in '--multiprocessing-fork':
```

Comment:
I've committed this update (I did it with --rebase so there are now 22 commits in this PR, not sure if this is your preferred method. If not, I can just create another PR)